lib/lndclient/LndClient.ts

```diff
@@ -340,15 +340,12 @@ class LndClient extends BaseClient {
       });
   }
 
-  /** Ends all subscriptions and reconnection attempts. */
-  public close = () => {
+  /** Ends all subscriptions attempts. */
```

Comment:
Maybe have this comment say something along the lines of "Lnd specific cleanup" (like you've done for Raiden) and move this general description to the base `close` method. Also "subscriptions attempts" looks like a typo here, maybe best to say something like "Ends all connections, subscriptions, and timers for for this client."